torch/nn/_functions/padding.py

```diff
@@ -42,21 +40,26 @@ def forward(self, input):
         c_output.copy_(c_input)
         return output
 
-    def backward(self, grad_output):
-        pad_l, pad_r, pad_t, pad_b = self.pad
+    @staticmethod
+    def backward(ctx, grad_output):
+        pad_l, pad_r, pad_t, pad_b = ctx.pad
+
+        grad_input = Variable(grad_output.data.new(ctx.input_size).zero_())
+        grad_input_slices = [slice(0, x + 1,) for x in ctx.input_size]
 
```

Comment:
This is necessary because of how I do the adjustments later (without checking if the slice start/stop is None.  I could move all the logic into the adjustments, but that seemed more complicated.